jaeger-spark-dependencies-cassandra/src/main/java/io/jaegertracing/spark/dependencies/cassandra/CassandraDependenciesJob.java

```diff
@@ -78,6 +78,12 @@ public static Builder builder() {
           System.getProperty("javax.net.ssl.trustStorePassword", ""));
       sparkProperties.put("spark.cassandra.connection.ssl.trustStore.path",
           System.getProperty("javax.net.ssl.trustStore", ""));
+      sparkProperties.put("spark.cassandra.connection.ssl.clientAuth.enabled",
+          Utils.getEnv("CASSANDRA_SSL_KEYSTORE", "").isEmpty() ? "false" : "true");
+      sparkProperties.put("spark.cassandra.connection.ssl.keyStore.path",
```

Comment:
Shouldn't this be configurable? The default is None so why setting it to an empty string?

https://github.com/datastax/spark-cassandra-connector/blob/master/doc/reference.md